### PR TITLE
Only run release check when a PR is opened or updated

### PR DESCRIPTION
**Pull request by jkimbo**
## Description

To prevent multiple release check comments.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

**Comment by BryceBeagle** [comment]
IIRC, those extra states were added so that the RELEASE file check would get triggered when a PR transitioned from Draft to Ready for Review. Potentially removing `opened` would be better?

File: .github/workflows/release-check.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Release file check
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review, review_requested]
+    types: [opened, synchronize, reopened]
```
Comment (patrick91) [inline .github/workflows/release-check.yml:5]: If we remove this then we don’t get the comment when the PR goes from draft to ready, what if debounced events on the bot instead? 🤔

**Comment by patrick91** [comment]
> IIRC, those extra states were added so that the RELEASE file check would get triggered when a PR transitioned from Draft to Ready for Review. Potentially removing `opened` would be better?

Oh, maybe that’s a better solution than mine ☺️

**Comment by jkimbo** [comment]
That makes sense. Will change.

**Comment by jkimbo** [comment]
Updated @patrick91 @BryceBeagle . I think we can still remove the "review_requested" event because the "ready_for_review" event should cover it.

**Review by patrick91 — APPROVED** [review]
Looks good to me 😊 let's see how it works!

**Pull request merged**